File: daq/slc/src/dqm/src/VerticalRange.cc
#include "dqm/VerticalRange.h"

using namespace Belle2;

VerticalRange::VerticalRange() : Range()
{

}

VerticalRange::VerticalRange(const std::string& name) : Range(name)
{

}

VerticalRange::VerticalRange(const std::string& name, double min, double max)
  : Range(name, min, max)
{

}

VerticalRange::~VerticalRange() throw()
{

}

std::string VerticalRange::getDataType() const throw()
{
  return "R1V";
}

File: daq/slc/extra/javalib/src/b2dqm/core/VerticalRange.java
package b2dqm.core;

public class VerticalRange extends Range {

	public VerticalRange(String name, double min, double max) {
		super(name, min, max);
	}

	public VerticalRange(String name, double min) {
		super(name, min, min);
	}

	public VerticalRange() {
		super("", 0, 0);
	}

	@Override
	public String getDataType() {
		return "R1V";
	}
	
}
